### PR TITLE
 Capstone Project 

### PR DESCRIPTION
**Pull request by odaialazzeh**
## HTML pages:
- [ ] I have created two pages for the project, the home and the about page. 
- [ ] Create an images folder with the necessary images for the design.

## CSS file:

- [ ] I have created style.css which contains the style of both pages and also added the styles for desktop and mobile versions.

## JavaScript file:

- [ ] I have created a javascript file with codes to open and close the mobile menu and dynamically implement the speaker's section when the pages to load. 

## Link to the video:

- [ ]  This is the link to the recorded video 
https://www.loom.com/share/d32fce7cca744904a5cb58488fe5e275

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,89 @@
...
+
+![home-page](images/home-page.png)
+
+
+### About Page 
+
+![about-page](images/about-page.png)
+
+## Go Live 
+https://odaialazzeh.github.io/capstone1-project/
```
Comment (ha-manel) [inline README.md]: - [ ] Kindly add `/home.html` to this link so it takes us to the home page of the website and not the README of the repo 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,89 @@
...
+
+## Acknowledgments
+
+A list of resources that i find helpful and would like to give credit to :
+
+- [Microverse ](https://www.microverse.org)
+- [GitHub ](https://www.github.com)
+- [Youtube ](https://www.youtube.com)
+- [W3schools ](https://www.w3schools.com)
+- [Stackoverflow ](https://stackoverflow.com)
```
Comment (ha-manel) [inline README.md:84]: - [ ] Kindly add credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) (and as indicated in the project requirements) 👍

File: about.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        <div class="footer-up">
+            <h2>Partner</h2>
+            <ul>
+                <li><img src="images/mozilla.svg" alt=""></li>
+                <li><img src="images/Google.svg" alt=""></li>
+                <li><img src="images/Naver.svg" alt=""></li>
+                <li><img src="images/DaumKakao.svg" alt=""></li>
+                <li><img src="images/Airbnb.svg" alt=""></li>
+            </ul>
+        </div>
```
Comment (ha-manel) [inline about.html]: - [ ] Kindly try to follow the design guidelines as much as possible. For the partners section (in the about page), it's hidden in the desktop version and it's visible in the mobile version.

## Your design:
image

## Design guidelines: 
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        <div class="footer-down">
+            <div class="left">
+            <h2><span>WD</span> Course </h2>
+            <h3>using HTML, CSS, and JavaScript</h3>
+            </div>
+            <div class="right">
+            <h3>2022 Web development course. </h3>
+            <h3>Some Rights Reserved</h3>
+            </div>
```
Comment (ha-manel) [inline about.html:106]: - [ ] For the footer part of the about page, kindly make it dark for desktop version and light in the mobile version, to match the design guidelines 💯 

## Your design:
image

## Design guidelines:
image

File: home.html [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+                <a href="#" id="campaign">WD Campaign</a>
+            </div>
+        </nav>
+    </header>
+
+    <section class="s1">
+        <div class="home">
+            <img src="images/Icon - Menu.svg" id="open-menu" alt="">
+             <h2>"Hello! Change world"</h2>
+             <p id="p-main">Web development course using HTML, CSS, and JavaScript</p>
```
Comment (ha-manel) [inline home.html]: - [ ] Kindly make this element an `h1` instead of `p` since this is the main header of the page 👍

File: about.html [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+            
+        </nav>
+    </header>
+
+    <section class="s1">
+        <div class="home-about">
+            <img src="images/Icon - Menu.svg" id="open-menu" alt="">
+             <h2>"Hello! Change world"</h2>
+             <p>Web development course using HTML, CSS, and JavaScript</p>
```
Comment (ha-manel) [inline about.html]: - [ ]  Kindly make this element an `h1` instead of `p` since this is the main header of the page 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @odaialazzeh 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️
* The website is looking awesome 💯 Keep rocking 🚀 
* Presentation video was very informational and professional ✨ 

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @odaialazzeh  👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**